graph/db/models/channel_edge_info.go

```diff
@@ -30,11 +30,9 @@ type ChannelEdgeInfo struct {
 
 	// NodeKey1Bytes is the raw public key of the first node.
 	NodeKey1Bytes [33]byte
-	nodeKey1      *btcec.PublicKey
```

Comment:
caching came from https://github.com/lightningnetwork/lnd/pull/706, I think it would be good to do a pathfinding benchmark and/or profile

Reply:
I didn't see any pathfinding or graph loading performance degradation. I'm not sure about allocations, but haven't spotted anything suspicious and the call sites seem not to be relevant for pathfinding at least.